src/ActiveInventory.cpp

```diff
@@ -180,3 +180,64 @@ std::shared_ptr<Item> ActiveInventory::searchById(unsigned long item_id) {
         return NULL;
     }
 }
+
+void ActiveInventory::printItems(std::string value) {
+
+    std::shared_ptr<Item> item;
+    std::map<unsigned long, std::shared_ptr<Item> >::iterator id_it;
```

Comment:
I know it's probably ingrained to do things this way because Dr. Plank hates modern C++ and he's the one who taught us, but you can use the ```auto``` keyword and avoid writing these long type names.

I'm fine with long type names, but I do think ```std::map<T1,T2>::iterator``` (and its related iterators) is a good place to use ```auto```.

Don't worry about changing this, just a suggestion for the future.